app/database_etl/test_adjustment_handler/Test function for estimating model.ipynb

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Personally don't think this jupyter notebook needs to be committed, it's pretty simple. I am gonna toss it